l10n/en-US/viewer.ftl

```diff
@@ -253,6 +253,8 @@ pdfjs-find-match-count-limit =
     }
 
 pdfjs-find-not-found = Phrase not found
+# Must contain an empty string.
+pdfjs-find-empty-message = {""}
```

Comment:
OK, but what do you suggest instead?

Currently we have, in my opinion, pretty ugly/hacky JS-code for this and it'd be nice to improve things here.

Reply:
I don't have a better solution than the existing code, but I'm not a developer (CC @eemeli in case he has one).

I understand the current code is hacky, but this is solving the issue by creating a different problem on the localization side, where it's much harder to control (and will create a poor experience for localizers translating the product).

While this approach is cleaner in terms of code, it moves the ugliness to localization, with the difference that it impacts

Reply:
@flodolo, on the fluent side when the id is the empty string, could we just return the empty string whatever the potential arguments are ?

Reply:
> @flodolo, on the fluent side when the id is the empty string, could we just return the empty string whatever the potential arguments are ?

No, because if a string is exposed for localization, the expectation is that it might need to change in some locales. For example, we have 4 of these in Firefox, and they're valid edge cases (i.e. no way to work around them in code) where this behavior needs to be respected ([example](https://transvision.flod.org/string/?entity=browser/browser/sanitize.ftl:clear-time-duration-suffix.value&repo=gecko_strings)).

As for this code, would it be possible to have 2 separate blocks (one empty with no Fluent attached, one with results) and toggle their visibility instead? Would that create flashing of content?

Reply:
> As for this code, would it be possible to have 2 separate blocks (one empty with no Fluent attached, one with results) and toggle their visibility instead? Would that create flashing of content?

Unfortunately that doesn't seem any better than what we currently have, implementation wise, so it appears that there's simply no "good" way to improve things here.